### ISSUE TITLE
[Accessibility][Keyboard - Actions]: 'Action' Button available under the node is not accessible by keyboard.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2062]
**User Experience:**
Keyboard dependent users will not be able to use the functionality of the control if the any control is not accessible via keyboard.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE
2. "Bot Framework Composer" homepage will gets opened.
3. Select "Dialog Actions" from the available Bots under 'Examples' and activate it.
4. Navigate the dialog 'Define conversation objective' then press 'Next'.
5. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
6. Now, the tree structure appears in the centre. Navigate to already created node.
7. Activate the 'Actions' button available below the name of the node.
8. Observe the issue.

**Actual:** 
"Action" button is not accessible by keyboard. On using mouse, the control are interactive as action is getting performed but its not accessible with keyboard.

**Note:**
Same issue observed with other button's which are present in the node similar to action button's such button's are in "Replacedialog" pane.

**Expected:**
The controls which are accessible with mouse must be accessible with keyboard. So, 'Action' button must be accessible with keyboard.

**MAS Impacted:** [MAS2.1.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B1F29ACF9-A3E7-414F-BD8D-7E3598D877BB%7D&file=MAS%202.1.1%20%E2%80%93%20Keyboard.docx&action=default&mobileredirect=true)

**Attachment:**
[MAS2.1.1_Action is not accesible with keyboard.zip](https://github.com/microsoft/BotFramework-Composer/files/4241323/MAS2.1.1_Action.is.not.accesible.with.keyboard.zip)

**Comment by alanlong9278** [comment]
@ashish315  @cwhitten  Keyboard move track logic is that we can move cursor between the whole node, `nodeMenu` and `edgeMenu`, so "Action" button is not accessible by keyboard because `Action` button is a part of step node, we can not move cursor into it with keyboard.

**Comment by cwhitten** [comment]
Begin a new dialog's dialog link is not in the tab order.

**Comment by ashish315** [comment]
@cwhitten As checked, still action button and other button's {that are like action button in the other nodes such as in replacedialog} are not accessible with keyboard.

**Comment by cwhitten** [comment]
@ashish315 please attach screenshots of what you refer to as "Action button" and "other buttons:

**Comment by ashish315** [comment]
Screenshots:

![image](https://user-images.githubusercontent.com/60913131/80144762-b5e92700-85cc-11ea-9caa-c8272b81cf53.png)
![image](https://user-images.githubusercontent.com/60913131/80144874-e761f280-85cc-11ea-9942-36ecbdb07df6.png)
![image](https://user-images.githubusercontent.com/60913131/80145004-17a99100-85cd-11ea-97db-7e32a687fe11.png)

**Comment by alanlong9278** [comment]
I will check it today @cwhitten  @ashish315

**Comment by ashish315** [comment]
Agree with @alanlong9278 So HCL is closing the issue.
